src/paginate.ts

```diff
@@ -220,6 +226,33 @@ export async function paginate<T extends ObjectLiteral>(
         }
     }
 
+    if (config.paginationType === PaginationType.CURSOR) {
+        const formatMessage = (msg: string) => {
+            logger.debug(msg)
+            throw new ServiceUnavailableException(msg)
+        }
+
+        if (config.cursorableColumns.length < 1) {
```

Comment:
add `if (!config.cursorableColumns ..)`

Reply:
```ts
if (!config.cursorableColumns?.length) {
    formatMessage("Missing required 'cursorableColumns' config.")
}
```
Thank you for your feedback :)
How about this one?